addons/web_editor/controllers/main.py

```diff
@@ -205,14 +205,13 @@ def remove(self, ids, **kwargs):
         removal_blocked_by = {}
 
         for attachment in Attachment.browse(ids):
-            # in-document URLs are html-escaped, a straight search will not
-            # find them
-            url = tools.html_escape(attachment.local_url)
-            views = Views.search([
-                "|",
-                ('arch_db', 'like', '"%s"' % url),
-                ('arch_db', 'like', "'%s'" % url)
```

Comment:
the delimiter `'` and `"` were there for a reason? if you remove `/image`, we don't want it to be blocked because there is a `src="/image_of_tiger"` in a view